conda/activate.py

```diff
@@ -798,7 +799,21 @@ def _get_environment_env_vars(self, prefix):
                     )
                     print(f"variable {dup} duplicated", file=sys.stderr)
                 env_vars.update(prefix_state_env_vars)
+        for reserved in RESERVED_ENV_VARS:
+            collect_reserved_vars = []
+            if reserved in env_vars.keys():
+                env_vars.pop(reserved)
+                collect_reserved_vars.append(reserved)
 
+            if collect_reserved_vars:
```

Comment:
I think this should be dedented so it isn't in the for loop.

Reply:
that's an unfortunate typo. Good catch!